sdk/core/azure-core-tracing-opentelemetry/pom.xml

```diff
@@ -103,6 +104,18 @@
           </compilerArgs>
         </configuration>
       </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>3.0.0-M3</version> <!-- {x-version-update;org.apache.maven.plugins:maven-surefire-plugin;external_dependency} -->
+        <configuration>
+          <systemPropertyVariables>
+            <!-- Parallel runs disabled due to concurrency issues causing failures in mocked tests. -->
+            <junit.jupiter.execution.parallel.enabled>false</junit.jupiter.execution.parallel.enabled>
```

Comment:
Is this required? Can mock tests be refactored to support parallel execution?

Reply:
yep, checking if that can be worked out.